### PR TITLE
refactor: init class ixlib using project manifest version

### PR DESCRIPTION
**Pull request by sherwinski**
Reduces repeat work/added maintenance of updating the project version in multiple places by extracting the package version from package.json

### CODE REVIEW

File: projects/ng-imgix/src/lib/ix-srcset.directive.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import { Directive, Input, ElementRef, OnInit } from '@angular/core';
 
+const PKG_VERSION = require('../../package.json').version;
```
Comment (frederickfogerty) [inline projects/ng-imgix/src/lib/ix-srcset.directive.ts:3]: Are you sure that this works in the browser? As I mentioned in Gatsby, this doesn't "just work" in the browser, it needs a relevant Webpack or Babel build step to transform this into the right value. If that's working here, then go ahead 👍

File: projects/ng-imgix/src/lib/ix-srcset.directive.spec.ts [hunk 2]
```diff
@@ -11,11 +11,13 @@ export class MockElementRef extends ElementRef {
 
 @Component({
   template: `
-  <img src="https://assets.imgix.net/examples/bluehat.jpg" />
-  <img src="https://assets.imgix.net/unsplash/motorbike.jpg" [ixSrcset]="{ minWidth: 800, maxWidth: 1200 }" />
-  <img src="https://assets.imgix.net/unsplash/bridge.jpg?w=500" ixSrcset />
-  <img src="https://assets.imgix.net/unsplash/raspberries.jpg?w=1000" [ixSrcset]="{ disableVariableQuality: true }" />
-  `
+  <div style="position: relative; overflow: hidden; display: inline-block; width: 10px; height: 10px;">
```
Comment (frederickfogerty) [inline projects/ng-imgix/src/lib/ix-srcset.directive.spec.ts:14]: Nit: is this meant to have a width and a height of 10 for 4 child images? 👀

**Review by frederickfogerty — CHANGES_REQUESTED** [review]

File: projects/ng-imgix/src/lib/ix-srcset.directive.ts (hunk 1 above)
Comment (sherwinski) [reply projects/ng-imgix/src/lib/ix-srcset.directive.ts:3]: I mean it works when I execute the tests (which run in the browser) but that might not mean it'll work on _all_ browsers? I'm willing to keep it for now and catch any edge cases through better testing -- hoping cypress will help there.

File: projects/ng-imgix/src/lib/ix-srcset.directive.spec.ts (hunk 2 above)
Comment (sherwinski) [reply projects/ng-imgix/src/lib/ix-srcset.directive.spec.ts:14]: This was more for cleanliness since some pretty large images are shown on screen when Karma runs. I believe the gatsby tests do the same thing as well.

File: projects/ng-imgix/src/lib/ix-srcset.directive.ts (hunk 1 above)
Comment (frederickfogerty) [reply projects/ng-imgix/src/lib/ix-srcset.directive.ts:3]: This isn't a browser thing, if it works in one it'll work in all of them. When you say "execute the tests" are these integration tests, or tests that run on the built library? One way to test if this works is if this `require` still exists in the dist file. If it does, I'd strongly suggest removing this.

Comment (sherwinski) [reply projects/ng-imgix/src/lib/ix-srcset.directive.ts:3]: Ah thanks for clarifying, I now see what you mean. Looks like tests were not running on the built version, so I'm now running into the issue you mentioned whenever I try to import the directive in a test app. I'm doing a bit more research, but I'll either open a new PR with a potential fix or just roll this back altogether. Thanks again @frederickfogerty

Comment (frederickfogerty) [reply projects/ng-imgix/src/lib/ix-srcset.directive.ts:3]: Resolved offline
